test/test-markovmodel.c

```diff
@@ -1,3 +1,4 @@
+#include <stdlib.h>
```

Comment:
Is this causing some sort of warning or error during the build? Or did something else prompt adding this?

Reply:
Yeah, there was an error. I think some simple type included in that file exists in `stdlib.h` and it wasn't indirectly being included on a platform I was building on (I think Ubuntu 18.04)